code/style.css

```diff
@@ -1,58 +1,176 @@
-  background: #ccc;
+  margin: 45px 100px;
   display: flex;
-  justify-content: space-around;
+  justify-content: space-evenly;
   align-items: center;
+  border-radius: 5px;
+}
+.start-view-cards {
+  font-size: 20px;
+  padding-top: 100px;
+  color: white;
 }
-
 .killer-deck {
-  width: 250px;
-  height: 400px;
-  background: #000;
+  width: 180px;
+  height: 350px;
+  background-color: #000;
   text-align: center;
-  padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  box-shadow: 5px 5px 5px #fa3a3a;
+  border: 1px solid white;
+  color: black;
+}
+
+.generating-text {
+  font-size: 24px;
+  color: white;
 }
 
 .weapon-deck {
-  width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+  width: 180px;
+  height: 350px;
+  background-color: #000;
   text-align: center;
-  padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  box-shadow: 5px 5px 5px #fa3a3a;
+  border: 1px solid white;
+  font-size: 24px;
 }
 
-.room-deck {
-  width: 250px;
-  height: 400px;
-  background: midnightblue;
+.weapon-info-wrapper {
+  padding-top: 75px;
+}
 
+.room-deck {
+  width: 180px;
+  height: 350px;
+  background-color: #000;
   text-align: center;
-  padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+  box-shadow: 5px 5px 5px #fa3a3a;
+  border: 1px solid white;
+  font-size: 24px;
+}
+
+.room-info-wrapper {
+  padding-top: 100px;
+}
+
+/*** LOADER ***/
+
+.loader-wrapper {
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+
+}
+
+.loader {
+  border: 16px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 16px solid #f83750;
+  width: 100px;
+  height: 100px;
+  -webkit-animation: spin 1s linear infinite; /* Safari */
+  animation: spin 1s linear infinite;
+  
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+  /*keyframes-selector {css-styles;}*/
+}
+
+/*** REVELATION SECTION ***/
+
+.revelation {
+  height: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 10px;
+}
+
+#mystery {
+  color: white;
+  /*font-family: 'Times New Roman', Times, serif;*/
+  font-size: 20px;
+  background-color: rgb(128, 128, 128, 0.9);
 }
 
 button {
   padding: 8px 16px;
-  background: crimson;
+  background: black;
+  color: white;
   border: none;
   font-size: 20px;
   border-radius: 8px;
   display: block;
   margin: 0 auto;
+  box-shadow: 2px 2px 2px white;
+  cursor: url(./assets/icons8-magnifying-glass-tilted-left-48.png),auto;
+  outline: none;
+  border: 1px solid white;
+}
+
+
+button:active {
+  transform: translateY(5px);
+  box-shadow: none;
+
+}
+
+/*** FOOTER ***/
+
+footer {
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  color: white;
+  background: black;
+  padding: 0 50px;
+}
+
+footer a {
+  text-decoration: none;
+  color: grey;
 }
```

Comment:
//I love your styling.😍 It really fits in with the theme of the project. Great use of positioning with flexbox. Next time, rememberg to syle only by class-name. Use id only for javascrict. Also consider giving the selctors a classname for styling. I had feedback in my last project that styling by selectors is "bad practice."